ggsql-R/R/engine.R

```diff
@@ -0,0 +1,48 @@
+
+ggsql_engine <- function(options) {
+  # Write a temporary file with the query
+  tmp <- basename(tempfile("ggsql", ".", paste0(".", "plt")))
```

Comment:
Nit: It doesn't really matter here, but in `ggsql-vscode` we set `.ggsql`/`.gsql` as the extension.

```suggestion
  tmp <- basename(tempfile("ggsql", ".", paste0(".", "gsql")))
```